projects/vdk-plugins/vdk-impala/tests/template_regression_test.py

```diff
@@ -0,0 +1,594 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import os
+import pathlib
+import re
+import time
+import unittest
+from unittest.mock import ANY
+from unittest.mock import DEFAULT
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import pytest
+from vdk.internal.core import errors
+from vdk.plugin.impala import impala_plugin
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import get_test_job_path
+
+
+@pytest.mark.skip(
```

Comment:
maybe the newly added content of `tests/jobs` needs to go to `tests/functional/jobs`, then the `TemplateRegressionTests` need to be started using `@pytest.mark.usefixtures("impala_service")` and 
```
@patch.dict(
        os.environ,
        {
            VDK_DB_DEFAULT_TYPE: "IMPALA",
            VDK_IMPALA_HOST: "localhost",
            VDK_IMPALA_PORT: "21050",
        },
    )
```
-> common to `test_impala_connection.py` ?

Reply:
I found out through a painstaking process that this annotation doesn't help at all, and all the default values are used instead( which are the same as the ones mentioned in your comment, should you try and change them the default ones get used). However the test impala instance included in a docker image is too old and all the templates are completely broken when running them locally.

Reply:
Could you please update the PR with this change, so we could reproduce the syntax errors, once the test is set to run with the Impala started in docker?